clients/src/main/java/org/apache/kafka/common/record/LogEntry.java

```diff
@@ -150,7 +150,7 @@
      * Get the first sequence number of this message set.
      * @return The first sequence number
      */
-    int firstSequence();
+    int baseSequence();
```

Comment:
haha. I see . you have gone the other way.. I think we can leave it as 'base'. the code is internally consistent, and that is more important. 

I think `firstTimestamp` and `firstSequence` are marginally clearer though. Either way, I am not attached.